### ISSUE TITLE
Component: Argument of type ... not assignable

### ISSUE THREAD

**Issue by doits** [neoclide/coc-vetur#11]
This might also be an issue with vetur (#10), but since I am using coc-vim I don't even know which version of vetur I am using, so I report it here first, maybe you can help.

When I import any component, for example like this:

```vue
<template>
  <div />
</template>

<script lang="ts">
import Vue from 'vue'

import Component from "@/components/Logo.vue"

export default Vue.extend({
  name: "bla",
  components: { Component }
})
</script>
```

It give me:

```
test.vue|10 col 27 error| [Vetur 2345] Argument of type '{ name: string; components: { Component: boolean; }; }' is not assignable to parameter of type 'ComponentOptions<Vue, DefaultData<Vue>, DefaultMethods<Vue>, DefaultComputed, PropsDefinition<Record<string, any>>, Record<string, any>>'. [E]
```

Somehow it thinks the component is of type `boolean`.

I'd report this to `vetur` directly (if this i a vetur problem), but how can I see the version of `vetur` being used?

**Comment by chemzqm** [comment]
Checkout package.json of this moudle or use output channel https://github.com/neoclide/coc.nvim/wiki/Debug-language-server#using-output-channel.

Use your own vetur by `vetur.dev.vlsPath` configuration.

**Comment by doits** [comment]
Thanks for your quick help! Output channel is of no help though:

```
Loaded bundled typescript@3.5.2.
Vetur initialized
```

Then I tried to install `vue-language-server` globally and point `coc-vetur` to it:

```json
{
  "vetur.dev.vlsPath": "/Users/username/.config/yarn/global/node_modules/vue-language-server"
}
```

Which gives ...

```
[coc.nvim] vetur server module "/Users/username/.config/yarn/global/node_modules/vue-language-server/dist/client/vueMain.js" not found!
```

Looks like the dist layout of `vue-language-server` was changed, it is the same issue as #9.

And https://github.com/neoclide/coc-vetur/blob/master/yarn.lock#L5621-L5622 is saying version `0.0.56` of `vue-language-server` is used by `coc-vetur`, is this correct? So maybe an update to `0.0.57` might fix some things, would test it out with `vetur.dev.vlsPath` option (if this worked).
